features/FEATURE_BLE/ble/pal/SecurityDb.h

```diff
@@ -147,19 +97,16 @@ class SecurityDb {
      * @param[in] handle valid connection handle
      * @return pointer to security entry, NULL if handle was invalid
      */
-    virtual SecurityEntry_t* get_entry(
-        connection_handle_t connection
+    virtual const SecurityDistributionFlags_t* get_distribution_flags(
```

Comment:
what? some copy and paste error?

Reply:
Not at all, it is the load of the distribution flags for the entry.

Reply:
the doxygen doesn't match